tutorials/7-cosmjs/4-with-keplr.md

```diff
@@ -306,11 +306,6 @@ getTestnetChainInfo = (): ChainInfo => ({
         coinGeckoId: "cosmos",
     },
     coinType: 118,
-    gasPriceStep: {
-        low: 1,
-        average: 1,
-        high: 1,
-    },
```

Comment:
Did this not work for you? 
I think this is a new Keplr thing. What's your Keplr version?

Reply:
Yes, this property no longer exists in `ChainInfo` type. I have 
```
    "@keplr-wallet/types": "^0.11.23"
```

Reply:
Ah I see here https://docs.keplr.app/api/suggest-chain.html that the `gasPriceStep` has been moved into `feeCurrencies[x]`. Can you put it there?

Reply:
Sure, done